pkg/template/servicebroker/test-scripts/bind.sh

```diff
@@ -2,8 +2,9 @@
 
 . shared.sh
 
-serviceUUID=${serviceUUID-$(oc get template cakephp-mysql-example -n openshift -o template --template '{{.metadata.uid}}')}
+serviceUUID=${serviceUUID-$(oc get template jenkins-ephemeral -n openshift -o template --template '{{.metadata.uid}}')}
```

Comment:
cakephp-mysql-example is not one of the default templates in the openshift namespace when bringing up via `oc cluster up`... I picked one that was